canonical-web-and-design/ubuntu.com
# OS specific promotion of tools on /download/server and /download/desktop

## Done

- Test of OS specific promotion of tools on /download/desktop and /download/server
- Trying to improve on the multipass strip and promote up to three items based on the user's OS

## Notes

- This is a test before we get to the larger work around the promotion and is wired up with utm data


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/desktop and http://0.0.0.0:8001/download/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Change your user agent (in the inspector his Ctrl+Shift+P, type 'network conditions' and change your user agent and see the items in the 'More ways to use Ubuntu everywhere' strip change

## Screenshots

### linux
![image](https://user-images.githubusercontent.com/441217/89413612-0706cc00-d721-11ea-98b7-3b4a5b46948f.png)


### windows
![image](https://user-images.githubusercontent.com/441217/89413655-184fd880-d721-11ea-8131-5f75525df282.png)

### mac
![image](https://user-images.githubusercontent.com/441217/89413693-2998e500-d721-11ea-8952-e7a9a5f6ce28.png)
